app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -219,18 +231,34 @@ open class HomeActivity : LocaleAwareAppCompatActivity() {
         themeManager.setActivityTheme(this)
         themeManager.applyStatusBarTheme(this)
     }
-
-    private fun setupToolbarAndNavigation() {
+    /**
+     * Returns the inflated [supportActionBar].
+     * Callers trying to get the [supportActionBar] from the Activity should call this.
+     * Name chosen to be more discoverable and avoid confusion with references to getToolbar
+     * We initially use a ViewStub for startup performance benefits, and
+     * after inflation, it becomes a Toolbar.
+     */
+    fun getSupportActionBarAndInflateIfNecessary(): ActionBar {
         // Add ids to this that we don't want to have a toolbar back button
```

Comment:
nit: this comment should go closer to where it originally was (i.e. inside the if block). tbh, I don't know what this comment means (and it sounds outdated) so... yeah. 😬